core/baseline-core-v1.0-psd01.md

```diff
@@ -819,65 +825,100 @@ The identifiers and identity utilized in a BPI MUST be controlled by its Princip
+
 *No mechanism should exist that would allow a single entity to deprive a Principal Owner of their identifier without their consent. This implies that:*
 
 #### **[CR7]<[D11]**	
 Once a namespace is created within the context of a DPKI, it MUST NOT be possible to destroy it.
 
+[[CR7]>[D11]](#cr7>d11) Testability: The DID method used must not allow for DID deletion to be enacted by anyone.
+
 #### **[CR8]<[D11]**	
 Namespaces in a DPKI utilized by a BPI MUST NOT contain blacklisting mechanisms that would allow anyone to invalidate identifiers that do not belong to them.
 
+[[CR8]>[D11]](#cr8>d11) Testability: The DID method used must not allow blacklisting mechanisms such as suppressing the delivery of DID documents through the universal resolver for specific DIDs.
+
 #### **[CR9]<[D11]**	
 The rules for registering and renewing identifiers in a DPKI utilized by a BPI MUST be transparent and expressed in simple terms.
 
+[[CR9]>[D11]](#cr9>d11) Testability: The DID method used in a BPI is registered in the DID registry and conformant to the [W3C DID Standard](https://www.w3.org/TR/did-core/).
+
 #### **[R36]**	
 If registration is used as security to an expiration policy, the Principal Owner MUST be explicitly and timely warned that failure to renew the registration on time could result in the Principal Owner losing control of the identifier.
 
+[[R36]](#r36) Testability: The DID method used within a BPI does not have an expiration policy.
+
 #### **[CR10]>[D11]**	
 Once set, namespace rules within a DPKI utilized by a BPI MUST NOT be altered to introduce any new restrictions for renewing or updating identifiers.
 
 *Otherwise, it would be possible to take control of identifiers away from Principal Owners without their consent.*
 
+[[CR10]>[D11]](#cr10>d11) Testability: The DID method used within a BPI has a specification where its rules around the mechanics of a DID method have never been altered. 
+
 #### **[CR11]>[D11]**	
 Within a DPKI utilized by a BPI, processes for renewing or updating identifiers MUST NOT be modified to introduce new restrictions for updating or renewing an identifier, once issued.
 
+[[CR11]>[D11]](#cr11>d11) Testability: The DID method used within a BPI has a specification for which the read, create, update, and delete DID operations have never been altered. 
```

Comment:
I'm not sure the requirement here is a good idea. What if the processes were insufficiently secure, and the change is one that makes them secure, and works?